styled.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module "emotion/styled" {
+  import { Component } from "react";
+
+  export type Styler = (
```

Comment:
This may need a name change. I plan on adding a `Style` component in the future and this might get confusing. If this is how styled-components looks though then I say leave it.